### ISSUE TITLE
Content Ideas for "First Steps with Git"

### ISSUE THREAD

**Issue by konradpa** [lnnrtwttkhn/version-control-book#166]
- [ ] Add section on .gitkeep
- [ ] Add customizing zsh (for example https://ohmyz.sh/)

**Comment by lnnrtwttkhn** [comment]
yeah, partly. don't worry about it. let's just keep both open, work through the TODOs and then we can close the issues.
